main.go

```diff
@@ -10,21 +10,39 @@ import (
 )
 
 type ProxyConfig struct {
+	// Port on whch the proxy is listening on
 	Port int `env:"PORT" envDefault:"3000"`
 
 	// Proxy target
 	RemoteAddress string `env:"REMOTE_ADDRESS"`
 
-	// InputValidation enables the proxy to validate the incoming proxy
-	// requests using
+	// InputValidation enables the proxy to validate / authenticate the
+	// incoming proxy requests using
 	InputValidation bool `env:"INPUT_VALIDATION"`
```

Comment:
```suggestion
	ShouldValidateRequests bool `env:"SHOULD_VALIDATE_REQUESTS"`
```

We try and adhere to a boolean naming convention of prefixing boolean variable names with:`is`, `does`, `should`, `has`, `was`, 'can', 'did', 'would' or 'are'

Reply:
👍